R/pool-methods.R

```diff
@@ -0,0 +1,89 @@
+#' @include pool.R
+NULL
+
+#' S4 class for compatibility with DBI methods
+#' @export
+setClass("Pool")
+
+## documented manually, with the Pool object
+#' @export
+poolCreate <- function(factory, ...,
+                       minSize = 1, maxSize = Inf,
+                       idleTimeout = 60000, validateTimeout = 2000,
+                       stateEnv = NULL) {
```

Comment:
Let's add `force(...)` as the first line here, especially if we remove the `checkDriver` call.

I'm slightly leery of having `...` here at all, due to the potential for param name collisions in the future, but I can live with it if you feel strongly about it.